src/main/java/com/mewna/catnip/entity/impl/interaction/ApplicationCommandInteractionDataOptionImpl.java

```diff
@@ -51,7 +51,8 @@
     private transient Catnip catnip;
     
     private String name;
-    private ApplicationCommandOptionType value;
+    private ApplicationCommandOptionType type;
+    private Object value;
```

Comment:
This shouldn't be stored as an object, it should be stored as the proper type. Wouldn't that be `ApplicationCommandOptionChoice`?

Reply:
No ApplicationCommandOptionChoice is only string, and integer. But the value data is always string and only the provided string ids can be fetched as role, channel or user objects.